school/lms/doctype/cohort_subgroup/cohort_subgroup.json

```diff
@@ -0,0 +1,99 @@
+{
+ "actions": [],
```

Comment:
If things with the subgroups are going well can we migrate this to LMS Batch before merging?

Reply:
I would prefer to do that after this PR is merged.